dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -276,6 +283,17 @@ def change_config(python, backend, cheatsheet, asciiart):
         console.print(f'[blue]Backend cached_value {backend}')
 
 
+@option_verbose
+@main.command(name='build-docs')
+@click.option('--docs-only', is_flag=True)
+@click.option('--spellcheck-only', is_flag=True)
+@click.option('--package-filter', type=click.Choice(AVAILABLE_PACKAGES))
```

Comment:
The option here should be "multiple" https://click.palletsprojects.com/en/8.0.x/options/#multiple-options. You should be able to specify:

--package-filter docker-stat --package-filter apache-airflow --package-filter apache-airflow-provider-google.

That will turn the "package_filter" into `List[str]`

Reply:
I get tuple of str not `List[str]` when given multiple `package_filter` as breeze command

Reply:
Oh yeah. Was not aurę od it's triple or.list :)